rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
+services. An adversary may attempt to enumerate IAM roles in order to determine whether a role exists or not before
+attempting to assume or hijack the discovered role.
+"""
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS IAM Assume Role Brute Force"
+references = [
+    "https://www.praetorian.com/blog/aws-iam-assume-role-vulnerabilities",
+    "https://rhinosecuritylabs.com/aws/assume-worst-aws-assume-role-enumeration/",
+]
+risk_score = 47
+rule_id = "ea248a02-bc47-4043-8e94-2885b19b2636"
+severity = "medium"
+tags = ["AWS", "Elastic"]
+type = "threshold"
+
+query = '''
+event.module:aws and event.dataset:aws.cloudtrail and event.provider:iam.amazonaws.com and event.action:UpdateAssumeRolePolicy and aws.cloudtrail.error_code:MalformedPolicyDocumentException and event.outcome:failure
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1110"
+name = "Brute Force"
+reference = "https://attack.mitre.org/techniques/T1110/"
+
+
+[rule.threat.tactic]
+id = "TA0006"
+name = "Credential Access"
+reference = "https://attack.mitre.org/tactics/TA0006/"
+
+[rule.threshold]
+value = 25
```

Comment:
I'm leaving this comment here for reviewers: I'm open to suggestions on what we should set this value to.

Reply:
I think if we have a hardcoded `rule.threshold`, we'll need to specify an interval too. 25 attempts in 5 minutes is different from 25 attempts in 60 minutes

Reply:
This applies to the other rule too. What _is_ the window for threshold rules? Is it the `from` to `to` window?

Reply:
@brokensound77 yes. This rule is currently using the default interval.

Reply:
sweet so `threshold.value/interval` makes sense, thanks 👍

Reply:
>I think if we have a hardcoded rule.threshold, we'll need to specify an interval too. 25 attempts in 5 minutes is different from 25 attempts in 60 minutes

the default is 5 mins so interval would only need to be specified if it is not the default (since we strip unneeded defaults which the API sets).

Going forward, I do think we will need to think about this and when it may make sense to do:
```sql
threshold.value = 25
interval = 5
```
vs
```sql
threshold.value = 5
interval = 1
```

Oh! Even more interestingly, signals on signals to do thresholds on thresholds, like `5 occurrences of 1 min threshold events in 10 mins`. 

Anyway, I think 25/5 is perfect in this instance

Reply:
I was just thinking about this again, due to the way AWS batches events over in 15m intervals, I'm not sure the default runtime/lookback of 5m/1m would be enough to cover all the events. Thoughts on adjusting the lookback time to `now-20m` @randomuserid?

Reply:
Can you add `from = "now-20m"` above the index @threat-punter. Then 👍

Reply:
Done. Cheers y'all